content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -353,6 +353,21 @@ prisma_pool_connections_open 1
 prisma_pool_connections_open 0
 
 # HELP prisma_client_queries_duration_histogram_ms Histogram of the duration of all executed Prisma Client queries in ms
+# HELP prisma_client_queries_wait_histogram_ms The wait time for a worker to get a connection.
+# TYPE prisma_client_queries_wait_histogram_ms histogram
+prisma_client_queries_wait_histogram_ms_bucket{le="0"} 0
+prisma_client_queries_wait_histogram_ms_bucket{le="1"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="5"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="10"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="50"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="100"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="500"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="1000"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="5000"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="50000"} 3
+prisma_client_queries_wait_histogram_ms_bucket{le="+Inf"} 3
+prisma_client_queries_wait_histogram_ms_sum 0.023208
+prisma_client_queries_wait_histogram_ms_count 3
```

Comment:
I think that is the wrong location. The `HELP` line above belongs to the metrics below.

Is that really how executing the command returns it!?

Reply:
Yes.